src/main/scala/uk/gov/homeoffice/drt/analytics/services/PaxModelStats.scala

```diff
@@ -34,7 +38,7 @@ object PaxModelStats {
 
   def sumActPctCapForDate(arrivals: Seq[Arrival]): Double = {
     val total = arrivals.map { a =>
-      (a.bestPcpPaxEstimate, a.MaxPax) match {
+      (a.bestPcpPaxEstimate(paxSourceOrderPreference), a.MaxPax) match {
```

Comment:
I've had to remove that hard coded order so that we can specify it on a port by port basis. This is so we can switch between ML figures and port feed figures when we need to